test/test_fsdp2/test_fsdp2_common.py

```diff
@@ -46,7 +47,10 @@ def check_parity_no_mp(
             ):
                 precompute_float8_dynamic_scale_for_fsdp(model)
 
-        test_cls.assertEqual(losses[0], losses[1])
+        if compile_transformer_block:
+            torch.testing.assert_close(losses[0], losses[1], atol=9.5e-2, rtol=9.5e-2)
```

Comment:
borrowed 9.5e-2 from `test_compile.py` https://github.com/pytorch-labs/float8_experimental/blob/main/test/test_compile.py#L62

Reply:
This seems kind of high 🤔  I wonder how this value was determined. Can we instead compare the ref as also compiling each transformer block (but without FSDP applied)?

Reply:
will try to switch the ref model to Float8Linear + torch.compiled

Reply:
after applying torch.compile to ref_model, we can achieve `atol/rtol=1e-4`. I can dig more as follow ups if we want to reach higher numeric parity like `1e-5`